src/12.datetimeformat.js

```diff
@@ -854,7 +854,14 @@ function CreateDateTimeParts(dateTimeFormat, x) {
                     fv = ''; // TODO
                     break;
 
-                    // TODO: Era
+                  case 'era':
+                    try {
+                      fv = resolveDateString(localeData, ca, 'eras', f, tm['[['+ p +']]']);
```

Comment:
Hmm. I tried same code with the above and they seems have something defference between `Intl.js` and `chrome`: https://gist.github.com/watilde/d0406a357ed8367d3cbd12f32dfdcf1e
I will try to take a look into some related things with this more.